cmd/krew/cmd/internal/setup_check_test.go

```diff
@@ -43,6 +45,35 @@ func TestIsBinDirInPATH_secondRun(t *testing.T) {
 	}
 }
 
+// TestIsBinDirInPATH_NonNormalized case when PATH content is not well normalized
+func TestIsBinDirInPATH_NonNormalized(t *testing.T) {
+	tempDir := testutil.NewTempDir(t)
+
+	// backup and restore initial PATH value
+	defer func(backupPath string) {
+		err := os.Setenv("PATH", backupPath)
+		if err != nil {
+			t.Fatalf("os.Setenv(\"PATH\", %s) failed with %v", backupPath, err)
```

Comment:
No need to change this. But if you want to not mess around with escaping, you can use backticks in Go. This is often more readable:
```suggestion
			t.Fatalf(`os.Setenv("PATH", %s) failed with %v`, backupPath, err)
```